src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -21,6 +21,8 @@ import { fetchAndInvokeInitializers } from './JSInitializers/JSInitializers.Serv
 let renderingFailed = false;
 let started = false;
 
+let _connection: HubConnection;
```

Comment:
We don't have an underscore prefix convention for the other variables in this file. Could we change it to `connection` for consistency?

I'm glad you did call it `_connection` for the initial review though as it makes it clearer which other usage sites are affected 😄